GTG/gtk/editor/taskview.py

```diff
@@ -463,6 +463,37 @@ def insert_tag(self, tag):
         line_mark = self.buff.create_mark("lastline", lastline, False)
         self.insert_at_mark(self.buff, line_mark, tag)
 
+    def remove_tag_editor(self, tagname):
+        '''
+        TODO(jakubbrindza): Finish works on this function which will
+        enable removing a tag from a task in task editor.
```

Comment:
I think you shouldn't include this function in this PR as it is not ready and it is not called anyway.